code/Modules/Items/Carriables/Item.cs

```diff
@@ -13,15 +13,18 @@ namespace ssl.Modules.Items.Carriables
     /// </summary>
     public partial class Item : Carriable, ISelectable
     {
+        public const string Tag = "Item";
```

Comment:
chore: This tag system is in lot of PR

Reply:
It's fixed, the other branches were base on this branch